tests/test.py

```diff
@@ -33,6 +34,28 @@ def test_python():
                "Pi is roughly 3",
                {"--py-files": py_file_url})
 
+
+@pytest.mark.skip(reason="must be run manually against a kerberized HDFS")
+def test_kerberos():
+    '''This test must be run manually against a kerberized HDFS cluster.
+    Instructions for setting one up are here:
+    https://docs.google.com/document/d/1lqlEIs98j1VsAyoEYnhYoaNmYylcoaBAwHpD29yKjU4.
+    You must set 'principal' and 'keytab' to the appropriate values,
+    and change 'krb5.conf' to the name of some text file you've
+    written to HDFS.
+
+    '''
+
+    principal = "nn/ip-10-0-2-134.us-west-2.compute.internal@LOCAL"
```

Comment:
So, the user has to modify this line in order to set the principal to their cluster?